README.md

```diff
@@ -62,11 +62,8 @@ Apache Polaris is built using Gradle with Java 21+ and Docker 27+.
 - `./gradlew assemble` - To skip tests.
 - `./gradlew test` - To run unit tests and integration tests.
 - `./gradlew polarisServerRun` - To run the Polaris server locally, with profile `prod`; the server 
```

Comment:
I'm not familiar Quarkus conventions, but "with profile `prod`" seems a bit odd to me, it is no way close to a `prod` deployment. Is "with the default profile" more accurate? We don't even have to mention it in that case.

Reply:
You are right. Rephrased.